orianna.py

```diff
@@ -410,12 +410,12 @@ def get_role_emoji(player):
         vision = ''
         vision_score = ''
         player_list.sort(key=lambda x: x.vision_score, reverse=True)
-        vision_score_list = list(p.vision_score for p in player_list)
-        percentile_25 = np.percentile(vision_score_list, 25)
-        percentile_50 = np.percentile(vision_score_list, 50)
-        percentile_75 = np.percentile(vision_score_list, 75)
```

Comment:
These are never called nor used. I commented them out in case they would be used in the future.